pagefaultgames/pokerogue
# Update splash-messages.ts [Localization]

## What are the changes?
A couple of splash messages have been fixed.

## Why am I doing these changes?
To make sure splash messages actually make sense and are funny.

## What did change?
Salvataggio Facoltativo -> Salvataggi facoltativi
Gioca con il 5x di Velocità -> Gioca con la velocità aumentata di 5 volte

### Screenshots/Videos



## How to test the changes?
Reload your pokerogue page so many times that you see all of the splash messages.

## Checklist
